<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AffineTransformWidget</class>
 <widget class="QWidget" name="AffineTransformWidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>297</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../resources/AffineTransform.xpm</normaloff>../../resources/AffineTransform.xpm</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="rotationHeaderLabel">
         <property name="text">
          <string>Rotation (deg):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="scalingHeaderLabel">
         <property name="text">
          <string>Scaling (%):</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="RotationLabelX">
         <property name="text">
          <string>x</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="rotationSpinBoxX">
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="ScalingLabelX">
         <property name="text">
          <string>x</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="scalingSpinBoxX">
         <property name="minimum">
          <double>10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="RotationLabelY">
         <property name="text">
          <string>y</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="rotationSpinBoxY">
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="ScalingLabelY">
         <property name="text">
          <string>y</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="scalingSpinBoxY">
         <property name="minimum">
          <double>10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="RotationLabelZ">
         <property name="text">
          <string>z</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="rotationSpinBoxZ">
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="ScalingLabelZ">
         <property name="text">
          <string>z</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="scalingSpinBoxZ">
         <property name="minimum">
          <double>10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="translationHeaderLabel">
         <property name="text">
          <string>Translation:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="shearHeaderLabel">
         <property name="text">
          <string>Shearing:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="TranslationLabelX">
         <property name="text">
          <string>x</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="translationSpinBoxX">
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="ShearLabelXY">
         <property name="text">
          <string>xy</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="shearSpinBoxXY">
         <property name="minimum">
          <double>-2.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="TranslationLabelY">
         <property name="text">
          <string>y</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="translationSpinBoxY">
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="ShearLabelXZ">
         <property name="text">
          <string>xz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="shearSpinBoxXZ">
         <property name="minimum">
          <double>-2.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="TranslationLabelZ">
         <property name="text">
          <string>z</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="translationSpinBoxZ">
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="ShearLabelYZ">
         <property name="text">
          <string>yz</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="shearSpinBoxYZ">
         <property name="minimum">
          <double>-2.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="resampleButton">
         <property name="text">
          <string>Resample</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="loadButton">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="transformMatrixLabel">
         <property name="text">
          <string>Transform</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="centreRotationRadioButton">
         <property name="text">
          <string>About centre</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableWidget" name="affineTransformDisplay">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="rowCount">
          <number>4</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>70</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>10</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>25</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>10</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
